Library/Homebrew/extend/os/linux/extend/ENV/std.rb

```diff
@@ -0,0 +1,10 @@
+module Stdenv
+  # Add libxml2 to CPPFLAGS
```

Comment:
Can remove this comment; the code is readable enough for it to be self-explanatory.